frontend.js

```diff
@@ -92,7 +92,7 @@ function removeAllTableRows(table) {
     }
 }
 
-/** Add an option to a table. */
+// Add an option to a table.
```

Comment:
The default way to format Javascript docstrings is with `/** ... */`

Reply:
Oh sorry, I will revert these. I had no idea these were docstrings. Makes sense now.

Reply:
Should be fixed now. I am not touching `frontend.js` here.